lms/djangoapps/support/static/support/jsx/entitlements/data/api/client.js

```diff
@@ -0,0 +1,54 @@
+import 'whatwg-fetch'; // fetch polyfill
+import Cookies from 'js-cookie';
+
+import endpoints from './endpoints';
```

Comment:
There might be added benefit importing it this way so that you're forced to specify `endpoints.entitlementList` later on (and make it abundantly clear that it's some endpoint value) but this file is not so big that importing the value like 

```javascript
import { entitlementList } from './endpoints';
```

would be hard to decipher.